src/classes/CRLP_Batch_Base.cls

```diff
@@ -118,6 +122,14 @@ public abstract class CRLP_Batch_Base {
         this.recordIds = recordIds;
     }
 
+    /*******************************************************************************************************
+     * @description Constructs the class - Rollup is for a specific set of Summary Object Id's
+     */
+    public CRLP_Batch_Base(CRLP_RollupProcessingOptions.RollupType jobType) {
+        this();
+        this.jobType = jobType;
```

Comment:
I noticed that on line 101 above, it passes `this.jobType` into the `CRLP_RollupBatch_SVC.getSummaryObjectType` method. However, in this constructor the `this.JobType` var isn't defined until after that call is made.